lib/libutee/include/rng_pta.h

```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2021, Foundries.io Ltd
+ */
+
+#ifndef __RNG_PTA__H
+#define __RNG_PTA__H
+
+#define PTA_RNG_UUID { 0x035a4479, 0xc369, 0x47f4, \
+		{ 0x94, 0x51, 0xc6, 0xfd, 0xff, 0x28, 0xad, 0x65 } }
+
+/*
+ * PTA_CMD_GET_ENTROPY - Get Entropy from RNG using crypto_rng_read()
+ *
+ * param[0] (inout memref) - Entropy buffer memory reference
+ * param[1] unused
+ * param[2] unused
+ * param[3] unused
```

Comment:
Please use the same format as in other PTA headers to describe parameters:
```
$ grep -E (value|memref)\[ lib/libutee/include/pta_*.h | head -10
lib/libutee/include/pta_device.h: * [out]     memref[0]        Array of device UUIDs
lib/libutee/include/pta_gprof.h: * [in/out] value[0].a: id
lib/libutee/include/pta_gprof.h: * [in]     memref[1]: profiling data
lib/libutee/include/pta_gprof.h: * [in/out] memref[0]: sampling buffer
lib/libutee/include/pta_gprof.h: * [in]     value[1].a: offset: the lowest PC value in the TA
lib/libutee/include/pta_gprof.h: * [in]     value[1].b: scale: histogram scaling factor
lib/libutee/include/pta_gprof.h: * [out] value[0].a: sampling frequency
lib/libutee/include/pta_invoke_tests.h: * [in]     memref[0]        source (non-secure shared memory)
lib/libutee/include/pta_invoke_tests.h: * [out]    memref[1]        destination (SDP secure memory)
lib/libutee/include/pta_invoke_tests.h: * [in/out]     memref[0]        SDP secure buffer to read from and write to
```

Reply:
ok